optin/reporting/test/models/database.spec.js

```diff
@@ -0,0 +1,28 @@
+const { getDatabase } = require('../../models/database');
+const { expect } = require('chai');
+const sqlite3 = require('sqlite3');
+
+const runBasicDatabaseInstanceTests = (database, databaseName) => {
+	it(`returns ${databaseName} sequelize instance`, () => {
+		// Assert:
+		expect(database.config.database).to.be.equal(databaseName);
+		expect(database.options.storage).to.be.equal(`./_data/${databaseName}.db`);
+		expect(database.options.dialectOptions.mode).to.be.equal(sqlite3.OPEN_READONLY);
+	});
+};
+
+describe('database', () => {
+	describe('completed db', () => {
+		// Arrange + Act:
+		const { completed } = getDatabase();
+
+		runBasicDatabaseInstanceTests(completed, 'completed');
+	});
+
+	describe('in_progress db', () => {
+		// Arrange + Act:
+		const { in_progress } = getDatabase();
+
+		runBasicDatabaseInstanceTests(in_progress, 'in_progress');
```

Comment:
since the Act is outside, this is not really running test but asserting, so consider renaming to something like assertOpenDatabaseProperties. additionally, it's more helpful to have assert here in test instead of helper.

Reply:
I see it was created a helper method to share with `in_progress` and `completed` DB instance.

But I think I have a better idea, I will refactor on this